index.html

```diff
@@ -20,53 +20,29 @@ <h4>Today, 18-01-2022</h4>
         <div class="flex-grid">
             <div class="col">
                 <h2>Add Story</h2>
-                <form action="/action_page.php">
```

Comment:
i have never seen form action like this, i would like an explanation of how php was used here

Reply:
this is deleted in a new pull request